modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/NumberInputTest.ts

```diff
@@ -44,6 +44,44 @@ describe('browser.tinymce.themes.silver.throbber.NumberInputTest', () => {
     TinyAssertions.assertContent(editor, '<p style="font-size: 16px;">a<span style="font-size: 16px;">b</span>c</p>');
   });
 
+  it('TINY-10129: plus and minus should not increase and decrease font size when the editor is readonly', () => {
+    const editor = hook.editor();
+    editor.setContent('<p style="font-size: 16px;">abc</p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
+    editor.mode.set('readonly');
+
+    TinyUiActions.clickOnToolbar(editor, '.tox-number-input .plus');
+    TinyAssertions.assertContent(editor, '<p style="font-size: 16px;">abc</p>');
+    assert.equal(Value.get(UiFinder.findIn(TinyUiActions.getUiRoot(editor), '.tox-number-input input').getOrDie() as any), '16px');
+
+    TinyUiActions.clickOnToolbar(editor, '.tox-number-input .minus');
+    TinyAssertions.assertContent(editor, '<p style="font-size: 16px;">abc</p>');
+  });
+
+  it('TINY-10129: Toolbar buttons should be properly disabled', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p style="font-size: 16px;">abc</p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
+
+    Waiter.pTryUntil('Disabled should not exist', () => UiFinder.exists(TinyUiActions.getUiRoot(editor), '.tox-number-input .plus[disabled="disabled"]'));
+    Waiter.pTryUntil('Disabled should not exist', () => UiFinder.exists(TinyUiActions.getUiRoot(editor), '.tox-number-input .minus[disabled="disabled"]'));
+    Waiter.pTryUntil('Disabled should not exist', () => UiFinder.exists(TinyUiActions.getUiRoot(editor), '.tox-number-input .input[disabled="disabled"]'));
+
+    editor.mode.set('readonly');
+
+    await UiFinder.pWaitFor('Plus button should be disabled', TinyUiActions.getUiRoot(editor), '.tox-number-input .plus[disabled="disabled"]');
+
+    await UiFinder.pWaitFor('Plus button should be disabled', TinyUiActions.getUiRoot(editor), '.tox-number-input .minus[disabled="disabled"]');
+
+    await UiFinder.pWaitFor('Plus button should be disabled', TinyUiActions.getUiRoot(editor), '.tox-number-input input[disabled="disabled"]');
+
+    editor.mode.set('design');
+
+    Waiter.pTryUntil('Disabled should not exist', () => UiFinder.exists(TinyUiActions.getUiRoot(editor), '.tox-number-input .plus[disabled="disabled"]'));
+    Waiter.pTryUntil('Disabled should not exist', () => UiFinder.exists(TinyUiActions.getUiRoot(editor), '.tox-number-input .minus[disabled="disabled"]'));
+    Waiter.pTryUntil('Disabled should not exist', () => UiFinder.exists(TinyUiActions.getUiRoot(editor), '.tox-number-input .input[disabled="disabled"]'));
```

Comment:
```suggestion
    await Waiter.pTryUntil('Disabled should not exist', () => UiFinder.notExists(TinyUiActions.getUiRoot(editor), '.tox-number-input .plus[disabled="disabled"]'));
    await Waiter.pTryUntil('Disabled should not exist', () => UiFinder.notExists(TinyUiActions.getUiRoot(editor), '.tox-number-input .minus[disabled="disabled"]'));
    await Waiter.pTryUntil('Disabled should not exist', () => UiFinder.notExists(TinyUiActions.getUiRoot(editor), '.tox-number-input .input[disabled="disabled"]'));
```

Reply:
Ops. At least the tests are working even after that fix.